`go generate ./...` requires `{sty,ts}lint`, which I don't have

Don't see anything about that in `CONTRIBUTING.md`, so this needs clarification or a fix.

```
13:28 $ make .bootstrap
make: '.bootstrap' is up to date.
13:28 $ go generate ./...
/Applications/Xcode.app/Contents/Developer/usr/bin/make -C .. -f cockroach/build/protobuf.mk
make[1]: Nothing to be done for `protos'.
make: Nothing to be done for `all'.
stylint -c .stylintrc styl
/bin/bash: stylint: command not found
make: *** [lint] Error 127
ui/ui.go:22: running "make": exit status 2
```

--- a/ui/Makefile
+++ b/ui/Makefile
@@ -83,7 +83,7 @@ all: lint $(GOBINDATA_TARGET) $(TEST_TARGET) $(PROTO_JSON_TARGET)
 	rm -f $(GOBINDATA_DEBUG_TARGET)
 
 .PHONY:
-lint:
+lint: npm.installed
 	stylint -c .stylintrc $(STYL_ROOT)
 	tslint -c $(TS_ROOT)/tslint.json $(shell find $(TS_ROOT) -name '*.ts')
 

